scripts/deployment/phase2-assets/collaterals/deploy_ctokenv3_usdc_collateral.ts

```diff
@@ -40,13 +40,12 @@ async function main() {
 
   const deployedCollateral: string[] = []
 
-  /********  Deploy CompoundV3 USDC - cUSDCv3  **************************/
+  /********  Deploy CompoundV3 USDC - cUSDCv3 **************************/
 
   const WrapperFactory: ContractFactory = await hre.ethers.getContractFactory('CusdcV3Wrapper')
-
```

Comment:
lets check for mainnet in this file (we have a separate usdbc file for base)

Reply:
Also its not just Mainnet, should be anything besides L2 Base (like tenderly should work as well, hh fork, etc)